web/ui/react-app/src/pages/graph/Graph.tsx

```diff
@@ -6,6 +6,9 @@ import { Legend } from './Legend';
 import { Metric, ExemplarData, QueryParams } from '../../types/types';
 import { isPresent } from '../../utils';
 import { normalizeData, getOptions, toHoverColor } from './GraphHelpers';
+import { Button } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Does this load it locally or via CDN's?

Reply:
Locally, we already use this icon in other places as well.